apps/src/lib/node/ledger/ethereum_node/mod.rs

```diff
@@ -132,11 +132,15 @@ pub mod eth_fullnode {
                     tracing::info!("Finished syncing");
                     break;
                 }
-                if let Err(e) = client.eth_syncing().await {
+                if let Err(error) = client.eth_syncing().await {
                     // This is very noisy and usually not interesting.
                     // Still can be very useful
-                    tracing::debug!("Error trying to connect to Geth: {:?}", e);
+                    tracing::debug!(
+                        ?error,
+                        "Couldn't connect to Geth, will retry"
+                    );
                 }
+                std::thread::sleep(std::time::Duration::from_secs(1));
```

Comment:
Can we have a `const SLEEP_DUR: Duration = ...` constant value here? Also, we're in an async function, this sleep will block other tasks. Let's switch to `tokio::time::sleep(SLEEP_DUR).await`

Reply:
The constant change is optional tbh. In my opinion it's better than leaving seemingly random params in a function/method call.

Reply:
Yeah a named constant is clearer, have changed to using `tokio::time::sleep` as well